pandas/_libs/src/datetime/np_datetime.c

```diff
@@ -28,6 +28,15 @@ This file is derived from NumPy 1.7. See NUMPY_LICENSE.txt
 #define PyInt_AsLong PyLong_AsLong
 #endif
 
+// Silence "implicit declaration of function" warnings
```

Comment:
shouldn't these be in the .h?

Reply:
No, that would make them public.  This particular fix will be superseded by #21962 anyway.